From gadiyar@ti.com  Fri Nov  6 14:56:05 2009
From: Anand Gadiyar <gadiyar@ti.com>
Date: Sat,  7 Nov 2009 01:16:32 +0530
Subject: USB: ehci: Allow EHCI to be built on OMAP3
Cc: Anand Gadiyar <gadiyar@ti.com>, Vikram Pandita <vikram.pandita@ti.com>, Ajay Kumar Gupta <ajay.gupta@ti.com>
Message-ID: <1257536793-15291-1-git-send-email-gadiyar@ti.com>


usb: ehci: Allow EHCI to be built on OMAP3

OMAP3 chips have a built-in EHCI controller.
The recently introduced omap ehci-hcd driver missed
out on selecting USB_ARCH_HAS_EHCI in Kconfig.

Without this, the driver cannot be built.

Signed-off-by: Anand Gadiyar <gadiyar@ti.com>
Cc: Vikram Pandita <vikram.pandita@ti.com>
Cc: Ajay Kumar Gupta <ajay.gupta@ti.com>
Acked-by: Felipe Balbi <felipe.balbi@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/Kconfig
+++ b/drivers/usb/Kconfig
@@ -61,6 +61,7 @@ config USB_ARCH_HAS_EHCI
 	default y if ARCH_W90X900
 	default y if ARCH_AT91SAM9G45
 	default y if ARCH_MXC
+	default y if ARCH_OMAP34XX
 	default PCI
 
 # ARM SA1111 chips have a non-PCI based "OHCI-compatible" USB host interface.
